controllers/ssp_controller.go

```diff
@@ -129,7 +129,7 @@ func (r *sspReconciler) setupController(mgr ctrl.Manager) error {
 func (r *sspReconciler) Reconcile(ctx context.Context, req ctrl.Request) (res ctrl.Result, err error) {
 	defer func() {
 		if err != nil {
-			common.SSPOperatorReconcilingProperly.Set(0)
+			common.SSPOperatorReconcileSucceeded.Set(0)
```

Comment:
Not really happy with this name either. What should this metric really express? That we reached phase `Deployed`?

Reply:
yes, seems like that. I think `kubevirt_ssp_operator_phase_deployed` makes sense.

Reply:
Seems like that, yes. But if you'd take a look at https://github.com/kubevirt/ssp-operator/blob/fc1d67c1b3ee193d02cab94149217525975c1f0c/controllers/ssp_controller.go#L612 you will see that we reach phase `Deployed` only if the reconcile succeeded (all available, no progressing or degraded), so this metric name makes sense to me.

Reply:
@0xFelix, @lyarwood WDYT?

Reply:
Let's stick with the proposed name for now.